test cases/common/62 string arithmetic/meson.build

```diff
@@ -1,16 +1,49 @@
 project('string arithmetic', 'c')
```

Comment:
We can add a minimum version here, and check in the file test.json whether a FeatureNew actually gets printed (and on which line of course).